### ISSUE TITLE
ANYOF searches variously failing and not

### ISSUE THREAD

**Issue by WValenti** [MathematicalMedicine/diverweb-issues#49]
Even verifying REGEX params coming out of diverweb is not fixing the semingly-random failures of ANYOF REGEX.

**Comment by WValenti** [comment]
Turns out this is due to the JSON_EXTRACT _requiring_ full quotes (which I started giving it), but then the SP was using the _quoted_ literals for comparisons, which would find nothing and then trigger the empty set, "info-free" error from the API code.  Talk about going in circles!  This is fixed by stripping full quotes from JSON_EXTRACTed literals while inserting them into the temporary table for the IN search.

Note that I left debris in the code.  The INSERTion of a GROUP_BY of the literals from the temporary table into Diag works fine, but should only happen when $debugsomethingorother allows.  Since there is no $debugaction macro, the INSERT is bare and always happens.
